.github/workflows/ruby_ci.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
       - name: Setup Bundler
         run: |
           echo "gem 'foreman_bootdisk', path: './foreman_bootdisk'" > bundler.d/foreman_bootdisk.local.rb
+          echo "gem 'foreman_fog_proxmox'" >> bundler.d/foreman_bootdisk.local.rb
```

Comment:
I would have expected this is not required, given you also edited `Gemfile`? (not blocking, just curious)

Reply:
Yeah, now it is not required.

Reply:
FYI: in our common CI we use `gemfile.d/*.rb`. For our reusable GHA it's [here](https://github.com/theforeman/actions/blob/a386c3ff5ce8e9a5faeefc75be67586eb03f9f3e/.github/workflows/foreman_plugin.yml#L75-L80) and Jenkins implements a similar thing.

Then in `Gemfile` you can use `eval_gemfile`:
```ruby
group :test do
  eval_gemfile 'gemfile.d/proxmox.rb'
end
```

This means you only specify `>= 0.14.2` once.

Reply:
@ekohl This also fails the rubocop as it cannot find the `gemfile.d/proxmox.rb`.

Reply:
> Added this change. But how will it find the `gemfile.d/proxmox.rb` as it's not there on foreman side as well as in foreman_bootdisk `gemfile.d` ?

Foreman doesn't care about `Gemfile`, so it shouldn't matter.

> @ekohl This also fails the rubocop as it cannot find the `gemfile.d/proxmox.rb`.

Because you didn't use `git add` so it's not in the repository.